src/spdx2/agent/spdx2.php

```diff
@@ -299,25 +365,39 @@ protected function toLicensesWithFiles(&$filesWithLicenses, $treeTableName)
       {
         $this->toLicensesWithFilesAdder($licensesWithFiles,$licenses['concluded'],$licenses['copyrights'],$fileId,$fullPath);
       }
-      elseif(!empty($licenses['scanner']))
+      elseif(!empty($licenses['scanner']) && count($licenses['scanner'])>0)
       {
-        $msgLicense = "NoLicenseConcluded (scanners found: " . implode(' or ',$licenses['scanner']). ")";
-        $this->toLicensesWithFilesAdder($licensesWithFiles,array($msgLicense),$licenses['copyrights'],$fileId,$fullPath);
+        $implodedLicenses = $this->implodeLicenses($licenses['scanner']);
+        if($licenses['isCleared'])
+        {
+          $msgLicense = "None (scanners found: " . $implodedLicenses . ")";
+        }
+        else
+        {
+          $msgLicense = "NoLicenseConcluded (scanners found: " . $implodedLicenses . ")";
+        }
       }
       else
       {
-        $msgLicense = "NoLicenseFound";
-        $this->toLicensesWithFilesAdder($licensesWithFiles,array($msgLicense),$licenses['copyrights'],$fileId,$fullPath);
+        if($licenses['isCleared'])
+        {
+          $msgLicense = "None";
+        }
+        else
+        {
+          $msgLicense = "NoLicenseConcluded";
+        }
       }
+      $this->toLicensesWithFilesAdder($licensesWithFiles,array($msgLicense),$licenses['copyrights'],$fileId,$fullPath);
```

Comment:
[Line 300](https://github.com/fossology/fossology/pull/633/files#diff-7bba19f12bab5e8b7e4b602c33f2e955L300) has to be updated as well, otherwise Files with concluded licenses are listed twice: one time with concluded license and second time with whatever $msgLicense was in previous loop.